tools/src/main/python/dlpx/virtualization/_internal/codegen.py

```diff
@@ -26,24 +26,6 @@
     'definitions': {}
 }
 
-#
-# The default snapshot params we currently support. This is so that in the
-# future when we want to support plugin author defined schemas for snapshot
-# params the upgrade case will be relatively simple.
-#
-
-SNAPSHOT_PARAMS_JSON = {
-    'snapshotParametersDefinition': {
-        'type': 'object',
-        'additionalProperties': False,
-        'properties': {
-            'resync': {
-                'type': 'boolean'
-            }
-        }
-    }
-}
```

Comment:
I think (not 100% sure) that `resync` should be a required property here. Like, the whole "snapshot parameters" object is optional, but if it exists, it should always have a `resync` property, right?

If I'm right, then this should have a line like `'required': ['resync']`.

Reply:
This is the snapshot params schema that we used to use for validation in our wrappers, so it should be safe to remove here. We never required `resync` before, but I think we should when we initialize the snapshot params schema in the future.

Reply:
yeah, I'm actually removing this haha. so while this was probably a bug that it wasn't required, since we weren't allowing the engine to actually specify their snapshot param, i think it was fine. Either way it's being removed so we should be good.